src/Executable.js

```diff
@@ -105,6 +111,63 @@ export default class Executable {
         return this;
     }
 
+    /**
+     * @param {number} minBackoff
+     */
+    setMinBackoff(minBackoff) {
+        if (this._maxBackoff != null && minBackoff >= this._maxBackoff) {
+            throw new Error(
+                "minBackoff cannot be larger than or equal to maxBackoff."
+            );
+        }
+        this._minBackoff = minBackoff;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get minBackoff() {
+        return this._minBackoff;
+    }
+
+    /**
+     * @param {number} maxBackoff
+     */
+    setMaxBackoff(maxBackoff) {
+        if (this._minBackoff != null && maxBackoff <= this._minBackoff) {
+            throw new Error(
+                "maxBackoff cannot be smaller than or equal to minBackoff."
+            );
+        }
+        this._maxBackoff = maxBackoff;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get maxBackoff() {
+        return this._maxBackoff;
+    }
+
+    /**
+     * @param {number} minBackoff
+     * @param {number} maxBackoff
+     */
+    setBackoff(minBackoff, maxBackoff) {
+        if (minBackoff <= maxBackoff) {
+            throw new Error("minBackoff cannot be larger than maxBackoff.");
+        }
+        this._minBackoff = minBackoff;
+        this._maxAttempts = maxBackoff;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get backoff() {
+        return this._maxBackoff, this._maxBackoff;
```

Comment:
The return value doesn't match the return type. Perhaps we should return an object
```javascript
{
    min: number;
    max: number;
}
```